experiments/react-native-uifabric/src/components/RadioGroup/RadioButton.tsx

```diff
@@ -0,0 +1,110 @@
+/** @jsx withSlots */
+'use strict';
+import * as React from 'react';
+import { View, Text, NativeSyntheticEvent } from 'react-native';
+import { radioButtonName, IRadioButtonType, IRadioButtonProps, IRadioButtonSlotProps, IRadioButtonRenderData } from './RadioButton.types';
+import { compose, IUseComposeStyling } from '@uifabricshared/foundation-compose';
+import { filterViewProps } from '../../utilities/RenderHelpers';
+import { ISlots, withSlots } from '@uifabricshared/foundation-composable';
+import { settings, radioButtonSelectActionLabel } from './RadioButton.settings';
+import { mergeSettings } from '@uifabricshared/foundation-settings';
+import { foregroundColorTokens, textTokens, borderTokens, backgroundColorTokens } from '../../tokens';
+import { useAsPressable } from '../../hooks';
+import { RadioGroupContext } from './RadioGroup';
+
+export const RadioButton = compose<IRadioButtonType>({
+  displayName: radioButtonName,
+
+  usePrepareProps: (userProps: IRadioButtonProps, useStyling: IUseComposeStyling<IRadioButtonType>) => {
+    const { content, buttonKey, disabled, ariaLabel, ...rest } = userProps;
+
+    // Grabs the context information from RadioGroup (currently selected button and client's onChange callback)
+    const info = React.useContext(RadioGroupContext);
+
+    const pressable = useAsPressable(rest);
+
+    // Used when creating accessibility properties in mergeSettings below
+    const onAccessibilityAction = React.useCallback((event: { nativeEvent: { actionName: any } }) => {
+      switch (event.nativeEvent.actionName) {
+        case 'Select':
+          info.onChange && info.onChange(buttonKey);
+          break;
+      }
+    }, []);
```

Comment:
Here you have to add `info` and `buttonKey` as a dependencies, because you use them inside the callback.

Reply:
Yes you're right, changing now

Reply:
Changed in newest iteration, resolving thread